api/jobs.yaml

```diff
@@ -116,6 +131,45 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    required:
+      - required
+    properties:
+      required:
+        type: boolean
+        description: Whether or not authentication is required
+      client_id:
+        type: string
+        description: OAuth 2.0 client ID
+      scopes:
+        type: array
+        description: OAuth 2.0 requested scopes
+        items:
+          type: string
+  CapabilitiesResponse:
+    description: Specification of the backend's capabilities
+    required:
+      - authentication
+    properties:
+      authentication:
+        $ref: '#/definitions/AuthenticationCapability'
+      display_fields:
+        type: object
+        description: Fields on QueryJobsResult returned from POST /jobs/query
+          populated on some or all jobs. The fields are mapped to their display
+          names, in order of importance. The UI will attempt to display as
+          many columns as possible given space constraints. Extended fields and
+          labels can be included, such as 'label.foo' or
+          'extended_fields.user_id'
+      label_query_fields:
+        type: object
+        description: Labels on QueryJobsRequest which are queryable, mapped to
+          their display names.
+      extended_query_fields:
+        type: object
+        description: Fields on ExtendedQueryFields which are queryable, mapped
+          to their display names.
```

Comment:
Actually, I don't think there's a reason to encode display names here. These fields are all hardcoded and have existing definitions; these shouldn't vary across backends. I'd just make this an array.

Reply:
Yeah I've been going back and forth on that, think I agree its better to use their hardcoded names. I made the change for label_query_fields too, let me know it you think there's a reason we should have those mapped still.